### PR TITLE
hid secrets

### PR DESCRIPTION
**Pull request by mattognibene**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 __pycache__
 db.sqlite3
 media
+secrets.py
```
Comment (samlowe106) [inline .gitignore:8]: We could also potentially do this using environment variables. It's technically the way you're "supposed" to do it, but it does require us to set up virtual environments. If you're interested we can go over it at the working session

File: klip/settings.py [hunk 2]
```diff
@@ -19,18 +20,14 @@
...
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+# TODO Limit hosts to our heroku instance
 ALLOWED_HOSTS = ['*']
 
-TWILIO_ACCOUNT_SID = "ACbb2cf8394f705901afac2dd8a0a4d4fd"
-TWILIO_AUTH_TOKEN = "f2f3c96578b98bcad50ab4c44e00172a"
-TWILIO_NUMBER = "+13522908811"
```
Comment (samlowe106) [inline klip/settings.py:32]: Have these keys been deactivated/replaced?

**Review by samlowe106 — COMMENTED** [review]
Looks good, just had some questions

File: klip/settings.py (hunk 2 above)
Comment (mattognibene) [reply klip/settings.py:32]: I think the auth token is old, will need it from dominik

File: .gitignore (hunk 1 above)
Comment (mattognibene) [reply .gitignore:8]: Yeah, definitely interested in looking at other ways to do this!

**Pull request merged**